packages/cashier/src/Components/Form/__tests__/account-transfer-form.spec.js

```diff
@@ -0,0 +1,451 @@
+        const props = mockProps();
+        props.selected_from = { currency: 'USD', is_crypto: true };
+        props.selected_to = { currency: 'USD', is_crypto: true };
+        props.transfer_fee = 2;
+
+        render(<AccountTransferForm {...props} />);
+
+        expect(
+            screen.getByText(
+                'We’ll charge a 2% transfer fee or 0 USD, whichever is higher, for transfers between your Deriv cryptocurrency accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 2%, is_mt_transfer and is_dxtrade_allowed', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+        props.transfer_fee = 2;
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed />);
+
+        expect(
+            screen.getByText(
+                'We’ll charge a 2% transfer fee or 0 USD, whichever is higher, for transfers between your Deriv cryptocurrency and DMT5 accounts and between your Deriv cryptocurrency and Deriv X accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 2%, is_mt_transfer, and is_dxtrade_allowed is false', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+        props.transfer_fee = 2;
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed={false} />);
+
+        expect(
+            screen.getByText(
+                'We’ll charge a 2% transfer fee or 0 USD, whichever is higher, for transfers between your Deriv cryptocurrency and DMT5 accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 2% and is_mt_transfer is false', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.transfer_fee = 2;
+
+        render(<AccountTransferForm {...props} />);
+
+        expect(
+            screen.getByText(
+                'We’ll charge a 2% transfer fee or 0 USD, whichever is higher, for transfers between your Deriv fiat and Deriv cryptocurrency accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 2% and is_mt_transfer is false', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.transfer_fee = null;
+
+        render(<AccountTransferForm {...props} />);
+
+        expect(screen.getByText('Please bear in mind that some transfers may not be possible.')).toBeInTheDocument();
+    });
```

Comment:
the name of the test case is the same as the previous one. is it ok?

Reply:
description is changed, thx for notice!